test/integration/garbagecollector/garbage_collector_test.go

```diff
@@ -453,18 +453,15 @@ func setupRCsPods(t *testing.T, gc *garbagecollector.GarbageCollector, clientSet
 		pod := newPod(podName, namespace, []metav1.OwnerReference{{UID: rc.ObjectMeta.UID, Name: rc.ObjectMeta.Name}})
 		createdPod, err := podClient.Create(context.TODO(), pod, metav1.CreateOptions{})
 		if err != nil {
-			t.Fatalf("Failed to create Pod: %v", err)
+			t.Errorf("Failed to create Pod: %v", err)
```

Comment:
It seems like `TestStressingCascadingDeletion` should have some way of receiving this err and doing a `Fatalf` instead